doc/scenario.md

```diff
@@ -675,7 +679,25 @@ Input facts (set by role):
 Input facts (set by config):
 
 - `cartridge_failover_promote_params` - promote leaders params. More details in
-  [rolling update doc](/doc/rolling_update.md).
+  [rolling update doc](/doc/rolling_update.md#leaders-promotion).
+
+### force_leaders
+
+Promotes leaders to current play hosts (instances specified in limit).
+More details in [rolling update doc](/doc/rolling_update.md).
```

Comment:
Let's use link to section with this variant of failover promote (Using `force_leaders` step)